### ISSUE TITLE
[LAB1] Απορία για το 4.5

### ISSUE THREAD

**Issue by DimLiak** [ntua-cvsp-lab/DSP_LabSupport#28]
όταν ζητάει να επιβεβαιώσουμε το αποτέλεσμα μας ακουστικά, τι ακριβώς θεέλετε να κάνουμε?

**Comment by cgaroufis** [comment]
Τέλος, αυτό που ζητείται εδώ είναι η προσεγγιστική επιβεβαίωση (ή όχι) του ρυθμού που έχετε εξάγει από το μουσικό κομμάτι, ακούγοντας το απόσπασμα που επιλέξατε (για να "εκτιμήσετε" ακουστικά το BPM, μία προσέγγιση θα ήταν να μετρήσετε τα beats του ~3 sec αποσπάσματος, και από εκεί να κάνετε extrapolate τα beats per minute)

Καλή συνέχεια,
Χρήστος

**Comment by DimLiak** [comment]
ως  beats θεωρούμε τα πικς του ήχου που θα υπάρχουν στο  κομματι του τραγουδιού?

**Comment by cgaroufis** [comment]
Ναι, τα οποία συνήθως συμβαδίζουν με τα "peaks" του κρουστικού περιεχομένου (drums, cymbals, etc) του μουσικού σήματος. 

Αν και αυτό ξεφεύγει από τους σκοπούς της άσκησης, ένα συχνό βήμα preprocessing σε αλγορίθμους tempo estimation αφορά στο διαχωρισμό του "ρυθμικού" μέρους του μουσικού σήματος (drums+bass) από το μελωδικό (φωνητικά, κιθάρα, πλήκτρα...)

Χρήστος

**Comment by DimLiak** [comment]
οκ νομιζω καταλαβα, ευχαριστω !
